infra/debian/compiler/docs/tflite-infer.1

```diff
@@ -0,0 +1,32 @@
+.TH TFLITE-INFER "1" "August 2022" "tflite-profile version 1.17.0" "User Commands"
```

Comment:
Please refer https://github.com/Samsung/ONE/issues/9194#issuecomment-1139494662 to generate this file. 

@mhs4670go , can you share the base rule of needs for this file?
that is, do we need doc file for tflite-infer ?

Reply:
This manpage is working on draft. It's not ready at all, but I'll be really grateful if you give me the base rule or something! :heart:

Reply:
There's no base rule. Just posting a comment that shows what the man page looks like when we type `man tflite-infer` in the bash would be helpful to review this file.

Reply:
@mhs4670go , thanks for the comment.
If the user wants to see the help them the user is curious about the command tool while using.
If the tool itself is not direct command for the user than we can think of providing the help can be optional (nice to have).
@yunjayh , is this `tflite-infer` a tool that a user uses normally? I think it would be a tool for ONE-vscode.

Reply:
> @yunjayh , is this `tflite-infer` a tool that a user uses normally? I think it would be a tool for ONE-vscode

It's not expected to use direct by user in cli command. However, as `one-infer` being a part of `one-cmds`, I thought the users will be curious about the `tflite-infer`'s command line usage.

Reply:
> as one-infer being a part of one-cmds

I think adding help for `one-infer` is enough as `one-infer` exist for the user.
- current document files are outdated: version and release year. we need to update them.
- adding `tflite-infer` is OK but this will increae one more file to maintain

But adding the file does not harm much so keeping this is OK.